packages/synapse-react-client/src/lib/containers/TextField.tsx

```diff
@@ -27,33 +28,49 @@ type TextFieldProps = Pick<
   | 'sx'
   | 'type'
   | 'value'
->
+> & { noWrapInFormControl?: boolean }
 
 /**
  * A styled text field built using MUI components and designed to match the Sage Design System (SDS) input fields.
  */
 export default function TextField(props: TextFieldProps) {
   const id = useId()
   const { palette } = useTheme()
+  const { noWrapInFormControl, ...rest } = props
+  const Wrapper = useMemo(
+    () =>
+      noWrapInFormControl
+        ? (props: React.PropsWithChildren<object>) => (
+            <React.Fragment>{props.children}</React.Fragment>
+          )
+        : (props: React.PropsWithChildren<object>) => (
+            <FormControl fullWidth sx={{ my: 1 }}>
```

Comment:
FormControl adds undesirable flex layout and margin to the input that was incompatible with the design.